project uploaded

Margocan opened the pull request:
confirmation

Margocan committed:
project uploaded

Margocan committed:
Uploading homework7

Margocan committed:
Readme added with comments about project

FiodarShtytsko commented on HW6/portf/portf/ViewController.swift:
```diff
@@ -0,0 +1,19 @@
+
+import UIKit
+
+class ViewController: UIViewController {
```
final везде

FiodarShtytsko commented on HW6/portf/portf/ViewController.swift:
```diff
@@ -0,0 +1,19 @@
+
+class ViewController: UIViewController {
+    
+    @IBAction private func pushDidTap () {
```
поменяй структуру методов расположения методв и переменных, пример есть в тг

FiodarShtytsko commented on HW6/portf/portf/ViewController.swift:
```diff
@@ -0,0 +1,19 @@
+class ViewController: UIViewController {
+    
+    @IBAction private func pushDidTap () {
+        let mainStoryboard = UIStoryboard (name: "Main", bundle: nil)
```
убрать magic number(это строки, числа, которые мы просто задаем без обозначения - "Main"), найти способ как можно работать без него

FiodarShtytsko commented:
Большой лейбл обрубается на третьем экране, следует поправить
<img width="280" alt="Screenshot 2024-01-11 at 13 17 24" src="https://github.com/Margocan/Homeworkmain/assets/134956650/f9ea9d26-63c7-4c8a-9439-dd22803f0cba">

Margocan committed:
Redo Homework6:
1) Change constrains ThirdVC main label "Hard skills"
2) Change to Final Class
3) Remove magic numbers (ViewController, MainStoryboard)
4) Changed structure of code

Margocan commented:
Solved

Margocan committed:
Homework7 in process
